haas/cli.py

```diff
@@ -401,6 +431,24 @@ def stop_console(node):
     do_delete(url)
 
 @cmd
+def make_initial_admin(username, password):
```

Comment:
Quoting henn (2016-02-23 14:10:00)

>    I think it'd also be good to see "user" in the name, and
>    create_initial_admin_user seems awkwardly long, so maybe
>    create_admin_user?

I think it's probably pretty clear even without `_user`, and I prefer
having the `initial` or something like it in there (again, encouraging
best practices) also given that the command should really only going to
be used once per setup, I don't think length is a big deal.  However, I
don't feel strongly about it; if you want me to change it to any of the
above I'll do so; let me know.

If `initial` is too long, we could also do `temp` or `tmp`.